### DISCUSSION TITLE
Select searchable and image upload missing styles

### DISCUSSION THREAD

**Discussion by joaonascimentodev** [filamentphp/filament#6782] [Help] [answered]
Hi, In a project that I make when I generate a form with a Select searchable and image upload they dont load styles.
Image: https://i.imgur.com/j2zb4Gn.png

Code generate form: 
` public static function form(Form $form): Form
    {
        return $form
            ->schema([

                Grid::make([
                    'default' => 1,
                    'sm' => 2,
                    'md' => 2,
                    'lg' => 4,
                    'xl' => 4,
                    '2xl' => 4,
                ])

                ->schema([

                    Forms\Components\Select::make('client_id')
                        ->label('Cliente')
                        ->required()
                        ->columnSpan(4)
                        ->options(Client::all()->pluck('name', 'id'))
                        ->searchable(),

                    Forms\Components\TextInput::make('invoice_ref')
                        ->label('Nº Fatura')
                        ->required()
                        ->maxLength(255)
                        ->columnSpan(2),

                    Forms\Components\DatePicker::make('date')
                        ->label('Data fatura')
                        ->required()
                        ->columnSpan(2),

                    
                    Forms\Components\TextInput::make('value')
                        ->label('Valor')
                        ->required()
                        ->columnSpan(2),

                    Forms\Components\FileUpload::make('invoice_file')
                        ->label('Fatura')
                        ->required()
                        ->directory('invoices')
                        ->visibility('public')
                        ->columnSpan(2),

                    Forms\Components\Toggle::make('status')
                        ->label('Pago?')
                        ->columnSpan(4),
                    
                    Forms\Components\FileUpload::make('receipt_file')
                        ->label('Recibo')
                        ->columnSpan(2),
                ])
            ]);
    }`

**Comment by danharrin** [comment, marked as answer]
Have you imported `module.esm.css` as per the installation instructions

**Reply by joaonascimentodev** [reply]
Hi, thanks for the answer. its solve. ;)
